care/facility/models/tests/test_patient.py

```diff
@@ -23,3 +27,22 @@ def test_disease_state_recovery_is_aliased_to_recovered(self):
         patient.refresh_from_db()
 
         self.assertEqual(patient.disease_status, DiseaseStatusEnum.RECOVERED.value)
+
+    def test_age_validation(self):
+        dist_admin = self.create_user("dist_admin", self.district, user_type=30)
+        sample_data = {
+            "facility": self.facility.external_id,
+            "blood_group": "AB+",
+            "gender": 1,
+            "year_of_birth": 2003,
```

Comment:
Test for invalid `year_of_birth` missing. 
Also only any one needs to be specified in a registration. Either year of birth or date of birth. Year of birth is ignored if date of birth is given (at model save).

```suggestion
```